stl/team_hi/adrian_hi/src/hello_world.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(){
+	std::cout<<"hello world"<<std::endl;
```

Comment:
indent.

Reply:
Thanks, I have modified the indentation as well as removed the bin file